app/models/upload.rb

```diff
@@ -148,8 +148,8 @@ def csv_file_headers
   def set_col_sep(first_line)
     self.col_sep = Settings.csv_upload.column_separators
                            .find { |column_separator| first_line.include?(column_separator) }
-
-    error_message = "Unable to determine column separator. #{Upload.valid_col_seps}"
+    valid_col_seps = Upload.valid_col_seps[:value].map { |cs| "\"#{cs}\"" }.join(' and ')
+    error_message = "Unable to determine column separator. #{valid_col_seps}"
```

Comment:
Does `valid_col_seps` represent valid column separators? The error message language doesn't make this obvious. Maybe something like,

```
Unable to determine column separator. Valid separators = [#{valid_col_seps}].
```